include/TransferElementAbstractor.h

```diff
@@ -142,7 +142,7 @@ namespace ChimeraTK {
      *
      * Note: Avoid using this in application code, since it will break the abstraction!
      */
-    const boost::shared_ptr<TransferElement>& getHighLevelImplElement() { return _impl; }
+    [[nodiscard]] boost::shared_ptr<TransferElement>& getHighLevelImplElement() { return _impl; }
```

Comment:
This needs to stay const, as we otherwise allow modifying the boost::shared_ptr in the abstractor (e.g. let it point to a different target). That is not what this function shall be used for.

Reply:
ok, I reverted that change (it was accidental and not required)